_data/community_instrumentations.yml

```diff
@@ -96,6 +96,15 @@
   sampling: "Yes"
   notes: Java 7 or higher, [SLF4J MDC support](https://github.com/Nike-Inc/wingtips#mdc_info) for auto-tagging all log messages with tracing info
 
+- language: JavaScript
+  library: >-
+    [appmetrics-zipkin](https://github.com/RuntimeTools/appmetrics-zipkin)
+  framework: Express, Koa
+  propagation: Http (B3)
+  transports: Http
+  sampling: "Yes"
+  notes: Provides zipkin support with addition of a single line of code
```

Comment:
question.. this leverages zipkin-js, right? so for example if someone wanted kafka, it would be a matter of making it auto-configurable in appmetrics-zipkin ya?

Reply:
yep, should be that straightforward